comparison.md

```diff
@@ -209,7 +209,7 @@ Help improve our documentation! If you find anything wrong or which could be imp
 				<td><strong>External Caching Systems (CDNs, Fastly)</strong></td>
 				<td>✅ Varnish automatically detects external Caching system and disables caching</td>
 				<td>✅</td>
-				<td>✅</td>
+				<td>✅ External caches respect Varnish/Drupal headers, Drupal modules available</td>
```

Comment:
I don't understand the part `External caches respect Varnish/Drupal headers`. With "external caching systems" we understand stuff like Fastly or a CDN, and obviously they respect the cache headers, but that is not a "feature" of Acquia :)
What we do at amazee.io is that our Varnish does not Cache a request again when it is sent from an external cache system like Fastly as it could lead into double caching.
can you elaborate here?